static/docs/commands-reference/import.md

```diff
@@ -21,28 +12,278 @@ Supported schemes:
       out            Output
 ```
 
+## Description
+
+DVC supports `.dvc` files which refer to an external data file, see
+[External Dependencies](/doc/user-guide/external-dependencies). In such a DVC
+file, the `deps` section will list the remote file specification and the `outs`
+section will list the local file name in the workspace. It records enough data
+from the remote file to enable DVC to efficiently check the remote file to
+determine if the local file is out of date.
+
+The `dvc import` command helps the user create such an external data dependency.
+
+It generates a DVC file listing in the `deps` section the resource named in
+the `url` parameter, and in the `outs` section the file named in the `out`
+parameter.
+
+DVC supports several types of remote files:
+
+Type | Discussion | URL format
+-----|------------|------------
+`local` | Local path | `/path/to/local/file`
+`s3` | Amazon S3 | `s3://mybucket/data.csv`
+`gs` | Google Storage | `gs://mybucket/data.csv`
+`ssh` | SSH server | `ssh://user@example.com:/path/to/data.csv`
+`hdfs` | HDFS | `hdfs://user@example.com/path/to/data.csv`
+`http` | HTTP to file with _strong ETag_ | `https://example.com/path/to/data.csv`
+
+In the _External Dependencies_ documentation, an alternative is demonstrated for
+each of these schemes.  Instead of:
+
+```dvc
+    $ dvc import https://example.com/path/to/data.csv data.csv
+```
+
+It is possible to instead run
+
+```dvc
+    $ dvc run -d https://example.com/path/to/data.csv \
+              -o data.csv \
+              wget https://example.com/path/to/data.csv -O data.csv
+```
+
+Both methods generate a DVC file with an external dependency. The `dvc import`
+command saves the user from using the command to copy files from each of the
+remote storage schemes, and from having to install CLI tools for each service.
+
+When DVC inspects a DVC file, one step is inspecting the dependencies to see if
+any have changed. A changed dependency will appear in the `dvc status` report,
+indicating the need to re-run the corresponding part of the pipeline. When DVC
+inspects an external dependency, it uses a method appropriate to that dependency
+to test its current status.
+
 ## Options
 
+* `--resume` - resume previously started download.
+
+* `-f`, `--file` - specify name of the DVC file it generates. It should be 
+  either `Dvcfile` or have a `.dvc` suffix (e.g. `data.dvc`) in order 
+  for `dvc` to be able to find it later. 
+
 * `-h`, `--help` - prints the usage/help message, and exit.
 
 * `-q`, `--quiet` - does not write anything to standard output. Exit with 0 if
   no problems arise, otherwise 1.
 
 * `-v`, `--verbose` - displays detailed tracing information.
 
-* `--resume` - resume previously started download.
+## Example: Initializing a workspace using a remote file
```

Comment:
name is too specific I think. Let's just simplify it to `Tracking a remote file`? And second example - `Detecting remote changes`?